src/dodal/devices/shutter.py

```diff
@@ -0,0 +1,44 @@
+from enum import Enum
+
+from bluesky.protocols import Movable
+from ophyd_async.core import AsyncStatus, StandardReadable, wait_for_value
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_w
+
+
+class ShutterSetState(str, Enum):
+    CLOSED = "close"
+    OPEN = "open"
+    RESET = "reset"
+
+
+class ShutterState(str, Enum):
+    CLOSED = ShutterSetState.CLOSED
+    OPEN = ShutterSetState.OPEN
+    OPENING = "opening"
+    CLOSING = "closing"
+    FAULT = "fault"
```

Comment:
Must:
RE: Tom's comment with the potential states, emphasis mine:

CTRL:

    **O**pen
    **Close**
    **R**eset

STA:

    **O**pen
    **Closed**
    **O**pening
    **C**losing
    **F**ault

I'd make an explicit new string for ShutterState.OPEN instead of reusing ShutterSetState.OPEN too.

Reply:
Open, Close, Reset are then commands not states.

ShutterCommand, ShutterState?

Reply:
capitalization - yeah, I must have missed it

Reply:
these are commands, but also states of the channel. IMO many devices have this 'state'